src/main/resources/webroot/management/openapi.yml

```diff
@@ -8,7 +8,7 @@ paths:
   /user:
     post:
       summary: Create a new user for this Cyface data collector
-      operationId: user
```

Comment:
Calling this addUser is not REST. In REST Operations are defined by the HTTP Methods: PUT, POST, GET, DELETE, etc. Everything else are just resources and should not be called as verbs.